.github/workflows/publish-images.yaml

```diff
@@ -0,0 +1,58 @@
+name: "Publish images"
+
+on:
+  push:
+    branches: [master]
+  release:
+    types: [published]
+
+jobs:
+  publish-containers:
+    name: Publish container images
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set env vars for the job
+        run: |
+          grep -v '\#' versions.txt | grep opentelemetry-collector | awk -F= '{print "OTELCOL_VERSION="$2}' >> $GITHUB_ENV
+          echo "VERSION_DATE=$(date -u +'%Y-%m-%dT%H:%M:%SZ')" >> $GITHUB_ENV
+          echo "VERSION=$(git describe --tags | sed 's/^v//')" >> $GITHUB_ENV
+
+      - name: Docker meta
+        id: docker_meta
+        uses: crazy-max/ghaction-docker-meta@v1
```

Comment:
crazy-max?  Should we be depending on someone named crazy-max?  :-)

Reply:
I had the exact thought. Unfortunately, the official docs for the Docker action recommends this one dynamic labels and tags.